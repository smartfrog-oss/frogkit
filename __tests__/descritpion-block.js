import { mount } from 'vue-test-utils'
import DescriptionBlock from '@/components/description-block'

describe('Description Block component', () => {

  it('should render component and match snapshot', () => {
    const wrapper = mount(DescriptionBlock, {propsData: {title: 'some title goes here'}})
    expect(wrapper.html()).toMatchSnapshot()
  })

})
