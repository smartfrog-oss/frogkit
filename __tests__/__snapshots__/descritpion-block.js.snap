// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Description Block component should render component and match snapshot 1`] = `
<section class="fk-description-block">
  <title size="md" color="primary" class="m-b-10">some title goes here</title>
  <div class="fk-description-block__content"></div>
</section>
`;
